automod/engine/ruleset.go

```diff
@@ -17,6 +18,7 @@ type RuleSet struct {
 	BlobRules         []BlobRuleFunc
 }
 
+// Executes all the various record-related rules. Only dispatches execution, does no other de-dupe or pre/post processing.
```

Comment:
fwiw, some (relatively popular) golang linters will complain if the docs don't start with literally the name of the method (per the golang convention that docs should read as sentences and thus begin with "Foobar is a [...]").

I don't particularly care about this, myself, but it is a common linter.

Reply:
good to know! I don't love that, but open to it in the future